org.eclipse.triquetrum.python.service/src/main/java/org/eclipse/triquetrum/python/service/PythonService.java

```diff
@@ -62,30 +61,27 @@ public static synchronized PythonService openConnection(final String pythonInter
 
     final PythonService service = new PythonService();
 
-    // Find the location of python_service.py and
-    // ensure org.eclipse.triquetrum.scisoft.python in PYTHONPATH
-    final Map<String, String> env = new HashMap<String, String>(System.getenv());
-    String pythonPath = env.get("PYTHONPATH");
-
-    StringBuilder pyBuf;
-    if (pythonPath == null) {
-      pyBuf = new StringBuilder();
-    } else {
-      pyBuf = new StringBuilder(pythonPath);
-      pyBuf.append(File.pathSeparatorChar);
-    }
     final int port = NetUtils.getFreePort(getServiceStartPort());
     String script = getSystemScriptsHome() + "/python_service_runscript.py";
 
-    service.command = new ManagedCommandline();
-    service.command.addArguments(new String[] { pythonInterpreter, "-u", script, String.valueOf(port), "-1" });
+    ProcessBuilder processBuilder = new ProcessBuilder(pythonInterpreter, "-u", script, String.valueOf(port), "-1");
 
-    env.put("PYTHONPATH", pyBuf.append(getSystemScriptsHome()).toString());
-    service.command.setEnv(env);
+    Map<String, String> environment = processBuilder.environment();
+    String pythonPath = environment.get("PYTHONPATH");
+    StringBuilder pyBuf = new StringBuilder();
+    if (pythonPath != null) {
+      pyBuf.append(pythonPath);
+      pyBuf.append(File.pathSeparatorChar);
+    }
+    pyBuf.append(getSystemScriptsHome());
+
+    environment.put("PYTHONPATH", pyBuf.toString());
 
-    // Currently log back python output directly to the log file.
-    service.command.setStreamLogsToLogging(true);
-    service.command.execute();
+    // TODO: log back python output directly to the log file.
```

Comment:
I have removed the whole cmdline package, but I have not replaced one part, the redirect of python output to the logging subsystem.